tox.ini

```
@@ -67,6 +67,8 @@ changedir =
 
 commands_pre =
   test: pip install .
+  test: bash -c 'curl -O -L https://github.com/googleinterns/cloud-operations-api-mock/raw/master/cmd/mock_server-x64-linux'
+  test: bash -c 'chmod +x mock_server-x64-linux'
```

I think a good choice would be to download this into [`{envbindir}`](https://tox.readthedocs.io/en/latest/config.html#substitutions-for-virtualenv-related-sections) which tox provides and already adds to `$PATH` when it runs tests.

Would be great to also check if the file is already downloaded and skip.

---

I also think the `bash -c` is not needed here
```suggestion
  test: curl -O -L https://github.com/googleinterns/cloud-operations-api-mock/raw/master/cmd/mock_server-x64-linux
  test: chmod +x mock_server-x64-linux
```

---

Yep, changed to download into `{envbindir}` and added a check to avoid re-downloading the same file. For the `bash -c` part, if I remove it, it says stuff like `WARNING: test command found but not installed in testenv. cmd: /usr/bin/chmod. DEPRECATION WARNING: this will be an error in tox 4 and above!`